recipes-connectivity/network-manager-applet/network-manager-applet-1.4.4/disable-auto-ethernet.patch

```diff
@@ -33,12 +33,33 @@
  	return TRUE;
  }
  
-@@ -238,8 +218,6 @@ ethernet_add_menu_item (NMDevice *device
+@@ -118,8 +98,6 @@ ethernet_add_menu_item (NMDevice *device
  
- 		if (g_slist_length (connections))
- 			add_connection_items (device, connections, carrier, active, ADD_INACTIVE, menu, applet);
+ 		if (connections->len)
+ 			applet_add_connection_items (device, connections, carrier, active, NMA_ADD_INACTIVE, menu, applet);
 -		else
--			add_default_connection_item (device, carrier, menu, applet);
+-			applet_add_default_connection_item (device, DEFAULT_ETHERNET_NAME, carrier, menu, applet);
  	}
+ }
+ 
+--- a/src/applet.c
++++ b/src/applet.c
+@@ -1354,8 +1354,7 @@ nma_menu_device_get_menu_item (NMDevice
+ 	}
+ 
+ 	if (!managed) {
+-		item = gtk_menu_item_new_with_label (_("device not managed"));
+-		gtk_widget_set_sensitive (item, FALSE);
++		return NULL;
+ 	}
```

Comment:
<nit> since the change resulted in a single line in the `if` statement, you could drop the brackets as well </nit>